Gruntfile.js

```diff
@@ -201,7 +202,7 @@ module.exports = function( grunt ) {
 		]
 	);
 
-	grunt.registerTask( "default", [ "dev", "test_fast", "compare_size" ] );
+	grunt.registerTask( "default", [ "dev", "eslint:dist", "test_fast", "compare_size" ] );
```

Comment:
Keep in mind though, that this is not terrible useful right now, since bunch of the rules are disabled for the build version cause sizzle does not follow jquery code style